lib/node/runner/api_block.go

```diff
@@ -108,13 +110,13 @@ func (nh NetworkHandlerNode) GetBlocksHandler(w http.ResponseWriter, r *http.Req
 	w.Header().Set("X-SEBAK-RESULT-COUNT", string(len(bs)))
 
 	for _, b := range bs {
```

Comment:
I have a question.  Why is the type of `bs` not `block.Block` but `interface{}`?   :->

Reply:
@anarcher Yes, `bs` is `[]interface{}`, because `bs` handles `block.Block` and `block.Header`.

Reply:
Humm,  Current Three calls about  get block from storage looks like to get `block.Block`..  Is it right?  https://github.com/bosnet/sebak/pull/421/files#diff-e805ed6b5e4b972f13e0939538ffb002R60
- https://github.com/bosnet/sebak/pull/421/files#diff-e805ed6b5e4b972f13e0939538ffb002R86
- https://github.com/bosnet/sebak/pull/421/files#diff-e805ed6b5e4b972f13e0939538ffb002R97

And render too.
``` 
nh.renderNodeItem(w, itemType, b.(*block.Block).Header)
nh.renderNodeItem(w, itemType, b.(*block.Block))
```

Reply:
@anarcher Haha, you pointed I missed. You right.